src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -72,6 +81,10 @@ public class Climber extends SubsystemBase {
   private boolean isClimbingStateSim;
 
   public Climber() {
+    currentLimit.StatorCurrentLimit = 90;
+    currentLimit.SupplyCurrentLimit = 70;
+    currentLimit.StatorCurrentLimitEnable = true;
+    currentLimit.SupplyCurrentLimitEnable = true;
```

Comment:
I think we'll want to set a lower current limit only when backing out (start with 10 Amps or less), and then set it higher again when climbing